### ISSUE TITLE
Alias

### ISSUE THREAD

**Issue by RichardLitt** [watson/geopkg#5]
Do you run `geopkg` instead of `npm version`? Do you have an alias in your bash_profile? Curious.

**Comment by watson** [comment]
@RichardLitt yes I run `geopkg version ...` instead of `npm version ...` - no alias. But I'm open to learning better ways to do it - but I've actually grown so accustomed to using this tool that I no longer even think of using `npm version`

**Comment by RichardLitt** [comment]
Well, it would be good if it was part of an alias, maybe, so that you don't actually obstruct userflow for npm users. That's a major source of friction for adoption, i think.

**Comment by watson** [comment]
@RichardLitt any idea of how to do that?

**Comment by RichardLitt** [comment]
I don't, short of forking npm. I do know that having `npmv` as an alias for `npm version` makes sense. That's easy to do:

``` sh
function npmv() {
  geopkg "$1"
}
```
